### PR TITLE
docs: fix file extension and missing export

### PR DESCRIPTION
**Pull request by JamBalaya56562**
## Description

1. Change `app/page.ts` to `app/page.tsx` at [Remote images section](https://nextjs.org/docs/app/getting-started/images-and-fonts#remote-images) in the [Images and Fonts docs](https://nextjs.org/docs/app/getting-started/images-and-fonts).
2. Use ESM export instead of CJS at [Customizing Sass options](https://nextjs.org/docs/app/getting-started/css-and-styling#customizing-sass-options) section to sync [cna template](https://github.com/vercel/next.js/blob/canary/packages/create-next-app/templates/app/js/next.config.mjs).
3. Add missing `export default nextConfig` line to `next.config.js` code and adjust next.config.ts format at [`styled-components`](https://nextjs.org/docs/app/getting-started/css-and-styling#styled-components) section in [Css and Styling](https://nextjs.org/docs/app/getting-started/css-and-styling) docs.

### Improving Documentation

- [x] Run `pnpm prettier-fix` to fix formatting issues before opening the PR.
- [x] Read the Docs Contribution Guide to ensure your contribution follows the docs guidelines: https://nextjs.org/docs/community/contribution-guide

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 20824ddeb375e8d334af3d66851efe0686bd601f

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

File: docs/01-app/01-getting-started/05-css-and-styling.mdx [hunk 1]
```diff
@@ -253,7 +253,7 @@ const nextConfig = {
   },
 }
 
-module.exports = nextConfig
+export default nextConfig
```
Comment (leerob) [inline docs/01-app/01-getting-started/05-css-and-styling.mdx:256]: This would need to be mjs, right?

Comment (JamBalaya56562) [reply docs/01-app/01-getting-started/05-css-and-styling.mdx:256]: Nice point out!
I checked Node.js [Determining module system](https://nodejs.org/api/packages.html#determining-module-system), it was good to change `js` to `mjs` file extension.

On the other hand, we can still use `js` when the nearest parent `package.json` file contains a top-level ["type"](https://nodejs.org/api/packages.html#type) field with a value of `"module"`.
Anyway, I have updated `mjs`.
Thanks.

File: docs/01-app/01-getting-started/05-css-and-styling.mdx [hunk 2]
```diff
@@ -392,13 +392,16 @@ const nextConfig: NextConfig = {
 export default nextConfig
 ```
 
-```js filename="next.config.js" switcher
+```mjs filename="next.config.mjs" switcher
```
Comment (leerob) [inline docs/01-app/01-getting-started/05-css-and-styling.mdx:395]:
```suggestion
```js filename="next.config.mjs" switcher
```

File: docs/01-app/01-getting-started/05-css-and-styling.mdx [hunk 3]
```diff
@@ -244,7 +244,7 @@ const nextConfig: NextConfig = {
...
```
Comment (leerob) [inline docs/01-app/01-getting-started/05-css-and-styling.mdx:247]:
```suggestion
```js filename="next.config.mjs" switcher
```

**Review by leerob — APPROVED** [review]

**Comment by JamBalaya56562** [comment]
Oh, my new commit interrupts CI needs your approve.
Could you approve again?

**Pull request merged**